exchanges/bitstamp/bitstamp.go

```diff
@@ -228,7 +228,10 @@ func (b *Bitstamp) GetBalance(ctx context.Context) (Balances, error) {
 	}
 	balances := make(map[string]Balance)
 	for k := range balance {
-		curr := k[0:3]
+		curr, _, found := strings.Cut(k, "_")
```

Comment:
It works specifically for what you're after, but it is incompatible with the fee section below.
You're splitting on the `_`, but then the code from lines 255-259 rely on appending a currency after it then splitting on the underscore. 

With no error checking on anything below (which is not on you), its hard to spot, but we end up with looking for a map entry of`zrxeurbtc_fee` when we're looking for `zrxeur_fee` to apply to the `eurFree` field. So this introduces some extra bugs

On this branch we get the following response:
```
          "BTC": {
          "Available": 0,
          "Balance": 0,
          "Reserved": 0,
          "WithdrawalFee": 0.0005,
          "BTCFee": 0,
          "USDFee": 0,
          "EURFee": 0
         },
         "BTCEUR": {
          "Available": 0,
          "Balance": 0,
          "Reserved": 0,
          "WithdrawalFee": 0,
          "BTCFee": 0,
          "USDFee": 0,
          "EURFee": 0
         },
```

on `master` we get:

```
  "BTC": {
          "Available": 0,
          "Balance": 0,
          "Reserved": 0,
          "WithdrawalFee": 0,
          "BTCFee": 0,
          "USDFee": 0.25,
          "EURFee": 0.25
         },
```

Reply:
No problem. Will fix.

Reply:
Just for fun: These values are 0 in live api always 😂

Reply:
Okay. So I've fixed this, but also uncovered that I'm pretty sure bitstamp pulled the rug on the v1 schema and returns 0 for all old *_fee fields.
It now just uses withdrawalFee.